src/components/application_manager/rpc_plugins/vehicle_info_plugin/test/commands/hmi/vi_get_vehicle_type_response_test.cc

```diff
@@ -0,0 +1,141 @@
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+
+#include "hmi/vi_get_vehicle_type_response.h"
+
+#include <string>
+
+#include "gtest/gtest.h"
+
+#include "application_manager/application.h"
+#include "application_manager/commands/commands_test.h"
+#include "application_manager/commands/response_from_hmi.h"
+#include "application_manager/mock_application_manager.h"
+#include "application_manager/mock_hmi_capabilities.h"
+#include "application_manager/mock_message_helper.h"
+#include "application_manager/policies/mock_policy_handler_interface.h"
+#include "application_manager/smart_object_keys.h"
+#include "smart_objects/smart_object.h"
+#include "vehicle_info_plugin/commands/vi_command_request_test.h"
+
+namespace test {
+namespace components {
+namespace commands_test {
+namespace hmi_commands_test {
+namespace vi_get_vehicle_type_response {
+
+using ::testing::_;
+using ::testing::NiceMock;
+using ::testing::Return;
+namespace am = ::application_manager;
+namespace strings = ::application_manager::strings;
+namespace hmi_response = am::hmi_response;
+namespace hmi_interface = ::application_manager::hmi_interface;
+using vehicle_info_plugin::commands::VIGetVehicleTypeResponse;
+
+typedef std::shared_ptr<VIGetVehicleTypeResponse> VIGetVehicleTypeResponsePtr;
+typedef NiceMock<
+    ::test::components::application_manager_test::MockHMICapabilities>
+    MockHMICapabilities;
+
+namespace {
+const uint32_t kConnectionKey = 2u;
+const std::string kVehicleType = "vehicle_type";
+}  // namespace
+
+class VIGetVehicleTypeResponseTest
+    : public VICommandRequestTest<CommandsTestMocks::kIsNice> {
+ public:
+  SmartObject capabilities_;
+};
+
+TEST_F(VIGetVehicleTypeResponseTest, RUN_SUCCESS) {
+  MessageSharedPtr command_msg(CreateMessage(smart_objects::SmartType_Map));
+  (*command_msg)[strings::params][hmi_response::vehicle_type] = kVehicleType;
+  (*command_msg)[strings::params][hmi_response::code] =
+      hmi_apis::Common_Result::SUCCESS;
+
+  VIGetVehicleTypeResponsePtr command =
+      CreateCommandVI<VIGetVehicleTypeResponse>(command_msg);
+
+  EXPECT_CALL(
+      mock_hmi_capabilities_,
+      set_vehicle_type(
+          (*command_msg)[strings::msg_params][hmi_response::vehicle_type]));
+  EXPECT_CALL(mock_hmi_capabilities_,
+              SaveCachedCapabilitiesToFile(hmi_interface::vehicle_info, _, _));
+  ASSERT_TRUE(command->Init());
+
+  command->Run();
+  EXPECT_TRUE((*command_msg)[am::strings::msg_params].keyExists(
+      hmi_response::vehicle_type));
+}
+TEST_F(VIGetVehicleTypeResponseTest, RUN_UNSUCCESS) {
```

Comment:
@ydementieiev 
- Add empty lines everywhere between test cases
- test case name is unclear. Why Run is unsuccessful?

Reply:
@AByzhynar [renamed](https://github.com/smartdevicelink/sdl_core/pull/3306/commits/23ce0bf712f3fcb162305d6a578433b880a1d473#diff-9005a1b4f41d513739d2200dadc2e8b3R106)